### PR TITLE
Feature: display group role, attach role and detach role

### LINKED ISSUE #76: Display roles assigned to a group
Blocked by #84 

On GroupPage, add RolesTable to display all roles of a group.

Actions are not to implemented in this task.

image

### LINKED ISSUE #78: Add a role to the group
Blocked by #84 

On GroupPage, create the button to "Add a role" (above the second table)

image

### LINKED ISSUE #80: Remove a role from a group
Blocked by #84 

On GroupPage, add an action on RolesTable.
Make actions optional and table don't do the result of action, only send the event that action are click on the parent.

You can call the props something like that: with-delete-action.

On delete event, display a confirmation popup and confirmation delete the group role association.

image

### PR DESCRIPTION
**Pull request by justinehell**
Resolves #76 
Resolves #78 
Resolves #80

### CODE REVIEW

File: src/services/GroupService.js [hunk 1]
```diff
@@ -68,3 +68,27 @@ export async function associateGroupAndUser(userLogin, groupId) {
...
   return api.delete(`/groups/${groupId}/users/${userLogin}`);
 }
+
+/**
+ * Associate group and role.
+ * @param {string} groupId - Group id.
+ * @param {string} roleId - Role id.
+ * @returns {Promise<object>} Promise with nothing on success otherwise an error.
+ */
+export async function associateGroupAndRole(groupId, roleId) {
```
Comment (Zorin95670) [inline src/services/GroupService.js:78]: move into RoleService

File: src/services/GroupService.js [hunk 2]
```diff
@@ -68,3 +68,27 @@ export async function associateGroupAndUser(userLogin, groupId) {
...
+  });
+}
+
+/**
+ * Dissociate group and role.
+ * @param {string} groupId - Group id.
+ * @param {string} roleId - Role id.
+ * @returns {Promise<object>} Promise with nothing on success otherwise an error.
+ */
+export async function dissociateGroupAndRole(groupId, roleId) {
```
Comment (Zorin95670) [inline src/services/GroupService.js:92]: move into RoleService

File: src/services/GroupService.js (hunk 1 above)
Comment (Zorin95670) [reply src/services/GroupService.js:78]: and rename to associateGroup(roleId, groupId)

Comment (justinehell) [reply src/services/GroupService.js]: done

File: src/services/GroupService.js (hunk 2 above)
Comment (justinehell) [reply src/services/GroupService.js]: done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**